### ISSUE TITLE
[millipede] documentation incomplete (Trac #1254)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1248]
the documentation in index.rst does not list all the options of millipede, e.g. the option ReadoutWindow is missing (there may be more missing, please check).

Migrated from https://code.icecube.wisc.edu/ticket/1254
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "the documentation in index.rst does not list all the options of millipede, e.g. the option ReadoutWindow is missing (there may be more missing, please check).", 
    "reporter": "hdembinski", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[millipede] documentation incomplete", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-20T19:13:30", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150824T20:19:37: 
* jbraun commented:

> In changeset 136809:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136809"
> Update docs.  Closes #1248
> ```


* jbraun changed status from "new" to "closed"
* jbraun changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
